weave

```diff
@@ -52,7 +52,7 @@ DNS_IMAGE=$BASE_DNS_IMAGE:$IMAGE_VERSION
 EXEC_IMAGE=$BASE_EXEC_IMAGE:$IMAGE_VERSION
 
 exec_remote() {
-    docker run -t --rm \
+    docker run --sig-proxy=true --rm \
```

Comment:
It is unnecessary, I included it for documentation. Would you still prefer it be removed?